nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/pandar_128e4x.hpp

```diff
@@ -91,7 +94,7 @@ class Pandar128E4X : public HesaiSensor<hesai_packet::Packet128E4X>
         channel_offset_ns = firing_time_offset_as1_ns_[channel_id];
       }
     }
-
+    
```

Comment:
Unnecessary newline?

Reply:
Addressed in [adf43fe](https://github.com/tier4/nebula/pull/126/commits/adf43fef5af5924675b60e317a6de3b59e7a9f06)